src/KingSystem/Event/evtInfoData.cpp

```diff
@@ -0,0 +1,72 @@
+#include "KingSystem/Event/evtInfoData.h"
+#include "KingSystem/Resource/resLoadRequest.h"
+#include "KingSystem/Utils/Byaml/Byaml.h"
+#include "basis/seadNew.h"
+#include "prim/seadSafeString.h"
+#include "resource/seadResource.h"
+
+namespace ksys::evt {
+
+SEAD_SINGLETON_DISPOSER_IMPL(InfoData)
+
+InfoData::InfoData() = default;
+
+void InfoData::init(sead::Heap* heap) {
+    this->doInit(heap, nullptr);
```

Comment:
```suggestion
    doInit(heap, nullptr);
```

The convention is to not use `this->` unless it is necessary. Nintendo uses a m prefix for member variables so manually writing `this` is mostly unnecessary.